code/game/gamemodes/wizard/artefact.dm

```diff
@@ -472,7 +472,7 @@ GLOBAL_LIST_EMPTY(multiverse)
 			if("cyborg")
 				if(!ismachine(M))
 					for(var/obj/item/organ/O in M.bodyparts)
-						O.robotize(make_tough = 1)
```

Comment:
Will not make robotics external limbs tough. They take the param make_tough. So either cast it and give the param to the casted version. Or add the param to the base proc